Compile.ps1

```diff
@@ -118,10 +119,12 @@ catch {
 Write-Progress -Activity "Validating" -Completed
 
 if ($run) {
-    try {
-        Start-Process -FilePath "pwsh" -ArgumentList "$workingdir\$scriptname"
-    } catch {
-        Start-Process -FilePath "powershell" -ArgumentList "$workingdir\$scriptname"
-    }
+    $script = "& '$workingdir\$scriptname' $arg"
+
+    $powershellcmd = if (Get-Command pwsh -ErrorAction SilentlyContinue) { "pwsh" } else { "powershell" }
+    $processCmd = if (Get-Command wt.exe -ErrorAction SilentlyContinue) { "wt.exe" } else { $powershellcmd }
+
+    Start-Process $processCmd -ArgumentList "$powershellcmd -NoProfile -Command $script"
```

Comment:
... so if you don't have `wt.exe` installed, it'll launch powershell (or pwsh).. and then that powershell will launch another powershell, which'll run the script?
This's way too nested, and I do agree that most devs working on this project work in Windows 11 (not 10), but I think there's room for improvements.

Reply:
I agree totally there, Chris liked using wt.exe if possible so I implemented it. Sure needs some work bc wt.exe and powershell work diffrently. Also this was already the case before. I think that may be a task for another PR tho..